INSTALL

```diff
@@ -13,11 +13,12 @@ Python XMP Toolkit
 ----------------------
 The short version of installation is::
 
-  python setup.py install
+  pip install python-xmp-toolkit
 
-or if you use pip::
+or if you use flit::
```

Comment:
I think this "or" is useless, because today everybody uses pip

Reply:
OK